src/screens/ParadeMapScreen/__snapshots__/Map.test.js.snap

```diff
@@ -580,10 +581,17 @@ exports[`Map component renders correctly 1`] = `
           },
         ]
       }
+      mapRef={
```

Comment:
I'm confused as to why mapRef is being passed to the AmenityMarkers props. Are we spreading props?

Reply:
This is my bad, I neglected to update the snaps after removing a ref I was passing that I was gonna use but ended up not needing to